<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>818</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>701</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="summary">
        <attribute name="title">
         <string>Summary</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="zone">
        <attribute name="title">
         <string>Zone</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="sensor">
        <attribute name="title">
         <string>Sensor</string>
        </attribute>
        <widget class="QWidget" name="horizontalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>691</width>
           <height>561</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QGroupBox" name="groupBoxSensor">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string/>
            </property>
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>241</width>
               <height>241</height>
              </rect>
             </property>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="labelSensorID">
                <property name="text">
                 <string>Sensor ID</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="comboBoxSensorID"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="labelValueType">
                <property name="text">
                 <string>Value Type</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="comboBoxSensorType"/>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="labelStart">
                <property name="text">
                 <string>Start</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QDateTimeEdit" name="dateTimeEdit"/>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="labelEnd">
                <property name="text">
                 <string>End</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QDateTimeEdit" name="dateTimeEdit_2"/>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="labelStepSize">
                <property name="text">
                 <string>Step Size</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QComboBox" name="comboBoxStepSize"/>
              </item>
              <item row="5" column="1">
               <widget class="QPushButton" name="pushButtonPlot">
                <property name="text">
                 <string>Plot</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="PlotWidget" name="sensorPlot"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="control_node">
        <attribute name="title">
         <string>Control Node</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
